README.md

```diff
@@ -3,3 +3,4 @@ A catalog of all projects in the Istio Ecosystem GH org
 
 * [istio-coredns-plugin](https://github.com/istio-ecosystem/istio-corends-plugin) - provide DNS resolution for hostnames in service entries
 * [cni](https://github.com/istio-ecosystem/cni) - Istio CNI plugin to inject sidecars by setting up the requisite networking configuration (iptables, etc.)
+* [wharf-multicluster-sync](https://github.com/istio-ecosystem/wharf-multicluster-sync) - create a proof-of-concept for user friendly multicluster Istio mesh configuration.
```

Comment:
nit: Just "a proof-of-concept for user friendly multicluster Istio mesh configuration."?

Reply:
today it is. :)  we can update it once we graduate from poc.

Reply:
Sorry, I meant remove the "create".